assets/js/components/NotFound/NotFound.jsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import Button from '@components/Button';
+import TrentoLogo from '../../../static/trento-icon.png';
+
+const NotFound = () => {
+  const navigate = useNavigate();
+
+  return (
+    <main className="bg-white relative overflow-hidden h-screen relative">
+      <div className="container mx-auto h-screen pt-32 md:pt-0 px-6 z-10 flex items-center justify-between">
+        <div className="container mx-auto px-6 flex flex-col-reverse lg:flex-row justify-between items-center relative">
+          <div className="w-full mb-16 md:mb-8 text-center lg:text-left">
+            <h1 className="font-light font-sans text-center lg:text-left text-5xl lg:text-8xl mt-12 md:mt-0 text-gray-700">
+              Sorry,<br></br>the page is in antoher castle.
```

Comment:
Is `antoher` some sort of joke that I don't get it? hehe

Reply:
super mario bros :)

Reply:
@arbulu89 now i get it it was a typo: "another"